_posts/en/newsletters/2020-04-22-newsletter.md

```diff
@@ -0,0 +1,185 @@
+  developer Olaoluwa Osuntokun's [introduction to the release
+  candidate][lnd rc intro] from the [LND engineering mailing list][].
+
+- [C-Lightning 0.8.2-rc1][c-lightning 0.8.2] is the first release
+  candidate for the next version of C-Lightning.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #17595][] adds support for [reproducibly building][topic
+  reproducible builds] Windows versions of Bitcoin Core using [GNU Guix][]
+  (pronounced "geeks"). The last remaining target platform for Guix reproducible
+  builds, macOS, has an open [draft PR][Bitcoin Core #17920].
+
+- [C-Lightning #3611][] adds a `keysend` plugin that allows a node to
+  securely receive [spontaneous payments][topic spontaneous
+  payments]---payments not preceded by the generation of an invoice.
+  These work by having the sender of a payment choose its payment
+  preimage (normally chosen by the receiver), derive its payment hash
+  (normally included in an invoice), encrypt the preimage to the
+  receiver's node pubkey, and send a payment with the encrypted data to
+  the receiver secured by the payment hash.  The receiver decrypts the
+  preimage and uses it to claim the payment like normal.  In order to
+  allow payment tracking, `lightningd` automatically creates an internal
+  invoice for the decrypted preimage before claiming a spontaneous
+  payment.  An obvious use for spontaneous payments is donations, but a
+  less obvious use is for sending chat messages with a payment such as
+  via the LND-compatible [WhatSat][] software and the
+  C-Lightning-compatible [noise][] plugin.
+
+- [C-Lightning #3623][] adds a minimal implementation (only available
+  with the configuration parameter `--enable-experimental-features`) for
+  spending payments using [blinded paths][].  As described in
+  [Newsletter #92][news92 blinded paths], blinded paths make it possible
+  to route a payment without the originator learning the destination's
+  network identity or the full path used.  This not only improves the
+  privacy of the origination and destination endpoints but also the
+  privacy of any unannounced nodes along the blinded paths.  The minimal
+  implementation in this PR is mainly designed for testing, such as with
+  a work-in-progress implementation being developed for Eclair.
+
+- [LND #4163][] adds a `version` RPC that returns information about the
+  LND server version and build flags.  This makes it easier for
+  applications to ensure they're compatible with the currently running
+  LND version.
+
+- [Rust-Lightning #441][] adds support for sending and receiving basic
+  [multipath payments][topic multipath payments] which allow a payment
```

Comment:
nit: I'm not sure if "which allow a payment to be split into multiple parts that may each be relayed through different channels." is necessary since we've covered multipath sever times before, including last week.